foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/internal/helper/WriteLockManager.java

```diff
@@ -66,7 +66,7 @@
 
     public static final int MAX_WAIT = 600000; //10 mins
     private static final int SIXTY_SECONDS = 60; //1 minute
-    private static final int SEVENTY_SECONDS = 70; //Once we reached 70s, we can assume we are in a bad state, so allow interruption for everything.
+    private static final int SEVENTY_SECONDS = 130; //Once we reached 130s, we can assume we are in a bad state, so allow interruption for everything.
```

Comment:
comment isn't accurate. Should be
// Once we reach SECONDS_BEFORE_HONOUR_INTERRUPT we consider the node in bad state for  SECONDS_IN_BAD_STATE seconds. During which time, we will honour all interrupts.